server/aws/lb.tf

```diff
@@ -57,6 +57,12 @@ resource "aws_lb" "covidshield_key_retrieval" {
   ]
   subnets = aws_subnet.covidshield_public.*.id
 
+  access_logs {
+      bucket  = "cbs-satellite-account-bucket005133826942"
```

Comment:
Would recommend injecting the account id using `data "aws_caller_identity" "current" {}`

Reply:
Good idea